From 0454861d3986a02904f7e5b101340b7eb9060598 Mon Sep 17 00:00:00 2001
From: Lance Albertson <lance@osuosl.org>
Date: Wed, 2 Oct 2019 20:11:43 -0700
Subject: [PATCH 3/3] Rename inspec gems to cinc-auditor gems

We need the ability to publish our own rubygems under our own namespace due to
how many of the other products (Cinc Client, Cinc Workstation, etc) rely on gems
for the omnibus installation. This provides a proper rename of all four gems
currently in use in InSpec.
---
 Gemfile                                                    |  4 ++--
 inspec-core.gemspec => cinc-auditor-core.gemspec           | 12 ++++++------
 inspec.gemspec => cinc-auditor.gemspec                     | 12 ++++++------
 inspec-bin/Gemfile                                         |  6 +++---
 .../{inspec-bin.gemspec => cinc-auditor-bin.gemspec}       | 12 ++++++------
 ...spec-core-bin.gemspec => cinc-auditor-core-bin.gemspec} | 14 +++++++-------
 6 files changed, 30 insertions(+), 30 deletions(-)
 rename inspec-core.gemspec => cinc-auditor-core.gemspec (78%)
 rename inspec.gemspec => cinc-auditor.gemspec (74%)
 rename inspec-bin/{inspec-bin.gemspec => cinc-auditor-bin.gemspec} (60%)
 rename inspec-bin/{inspec-core-bin.gemspec => cinc-auditor-core-bin.gemspec} (57%)

diff --git a/Gemfile b/Gemfile
index a590218d..bb487f85 100644
--- a/Gemfile
+++ b/Gemfile
@@ -1,13 +1,13 @@
 source "https://rubygems.org"
 
-gem "inspec", path: "."
+gem "cinc-auditor", path: "."
 
 # This dependency is NOT used for normal gem deployment
 # - instead, inspec-bin gemspec-depends on inspec
 #
 # However, AppBundler requires a top-level Gemfile.lock with inspec-bin
 # in it in order to package the executable. Hence the odd backwards dependency.
-gem "inspec-bin", path: "./inspec-bin"
+gem "cinc-auditor-bin", path: "./inspec-bin"
 
 gem "ffi", ">= 1.9.14"
 
diff --git a/inspec-core.gemspec b/cinc-auditor-core.gemspec
similarity index 78%
rename from inspec-core.gemspec
rename to cinc-auditor-core.gemspec
index ee708fd5..3929896e 100644
--- a/inspec-core.gemspec
+++ b/cinc-auditor-core.gemspec
@@ -3,13 +3,13 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "inspec/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "inspec-core"
+  spec.name          = "cinc-auditor-core"
   spec.version       = Inspec::VERSION
-  spec.authors       = ["Dominik Richter"]
-  spec.email         = ["dominik.richter@gmail.com"]
-  spec.summary       = "Just InSpec"
-  spec.description   = "Core InSpec, local support only. See `inspec` for full support."
-  spec.homepage      = "https://github.com/chef/inspec"
+  spec.authors       = ["Dominik Richter", "Cinc Project"]
+  spec.email         = ["dominik.richter@gmail.com", "cinc-auditor@cinc.sh"]
+  spec.summary       = "Just Cinc Auditor"
+  spec.description   = "Core Cinc Auditor (a community distribution of InSpec), local support only. See `cinc-auditor` for full support."
+  spec.homepage      = "https://gitlab.com/cinc-project/auditor"
   spec.license       = "Apache-2.0"
 
   spec.files = %w{README.md LICENSE} + Dir.glob("{bin,lib,etc}/**/*", File::FNM_DOTMATCH)
diff --git a/inspec.gemspec b/cinc-auditor.gemspec
similarity index 74%
rename from inspec.gemspec
rename to cinc-auditor.gemspec
index 6041d325..f5148979 100644
--- a/inspec.gemspec
+++ b/cinc-auditor.gemspec
@@ -3,17 +3,17 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "inspec/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "inspec"
+  spec.name          = "cinc-auditor"
   spec.version       = Inspec::VERSION
-  spec.authors       = ["Chef InSpec Team"]
-  spec.email         = ["inspec@chef.io"]
+  spec.authors       = ["Chef InSpec Team", "Cinc Project"]
+  spec.email         = ["inspec@chef.io", "cinc-auditor@cinc.sh"]
   spec.summary       = "Infrastructure and compliance testing."
-  spec.description   = "InSpec provides a framework for creating end-to-end infrastructure tests. You can use it for integration or even compliance testing. Create fully portable test profiles and use them in your workflow to ensure stability and security. Integrate InSpec in your change lifecycle for local testing, CI/CD, and deployment verification."
-  spec.homepage      = "https://github.com/inspec/inspec"
+  spec.description   = "Cinc Auditor (a community distribution of InSpec) provides a framework for creating end-to-end infrastructure tests. You can use it for integration or even compliance testing. Create fully portable test profiles and use them in your workflow to ensure stability and security. Integrate InSpec in your change lifecycle for local testing, CI/CD, and deployment verification."
+  spec.homepage      = "https://gitlab.com/cinc-project/auditor"
   spec.license       = "Apache-2.0"
 
   # the gemfile and gemspec are necessary for appbundler so don't remove it
-  spec.files = %w{Gemfile inspec.gemspec README.md LICENSE} + Dir.glob(
+  spec.files = %w{Gemfile cinc-auditor.gemspec README.md LICENSE} + Dir.glob(
     "{bin,lib,etc}/**/*", File::FNM_DOTMATCH
   ).reject { |f| File.directory?(f) }
 
diff --git a/inspec-bin/Gemfile b/inspec-bin/Gemfile
index 4ec425ac..a70e125a 100644
--- a/inspec-bin/Gemfile
+++ b/inspec-bin/Gemfile
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
-main_gemspec = File.expand_path("../inspec-bin.gemspec", __FILE__)
+main_gemspec = File.expand_path("../cinc-auditor-bin.gemspec", __FILE__)
 if File.exist?(main_gemspec)
-  gemspec name: "inspec-bin"
+  gemspec name: "cinc-auditor-bin"
 else
-  gemspec name: "inspec-core-bin"
+  gemspec name: "cinc-auditor-core-bin"
 end
diff --git a/inspec-bin/inspec-bin.gemspec b/inspec-bin/cinc-auditor-bin.gemspec
similarity index 60%
rename from inspec-bin/inspec-bin.gemspec
rename to inspec-bin/cinc-auditor-bin.gemspec
index 0a387ea6..beb16298 100644
--- a/inspec-bin/inspec-bin.gemspec
+++ b/inspec-bin/cinc-auditor-bin.gemspec
@@ -3,19 +3,19 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "inspec-bin/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "inspec-bin"
+  spec.name          = "cinc-auditor-bin"
   spec.version       = InspecBin::VERSION
-  spec.authors       = ["Chef InSpec Core Engineering "]
-  spec.email         = ["inspec@chef.io"]
+  spec.authors       = ["Chef InSpec Core Engineering", "Cinc Project"]
+  spec.email         = ["inspec@chef.io", "cinc-auditor@cinc.sh"]
   spec.summary       = "Infrastructure and compliance testing."
-  spec.description   = "InSpec executable for inspec gem. Use of this executable may require accepting a license agreement."
-  spec.homepage      = "https://github.com/inspec/inspec/tree/master/inspec-bin"
+  spec.description   = "Cinc Auditor (a community distribution of InSpec) executable for cinc-auditor gem."
+  spec.homepage      = "https://gitlab.com/cinc-project/auditor"
   spec.license       = "Apache-2.0"
 
   spec.require_paths = ["lib"]
   spec.required_ruby_version = ">= 2.4"
 
-  spec.add_dependency "inspec", "= #{InspecBin::VERSION}"
+  spec.add_dependency "cinc-auditor", "= #{InspecBin::VERSION}"
   spec.add_development_dependency "rake"
 
   spec.files = %w{README.md LICENSE Gemfile} + Dir.glob("*.gemspec") +
diff --git a/inspec-bin/inspec-core-bin.gemspec b/inspec-bin/cinc-auditor-core-bin.gemspec
similarity index 57%
rename from inspec-bin/inspec-core-bin.gemspec
rename to inspec-bin/cinc-auditor-core-bin.gemspec
index f791be2a..3444b9d4 100644
--- a/inspec-bin/inspec-core-bin.gemspec
+++ b/inspec-bin/cinc-auditor-core-bin.gemspec
@@ -6,22 +6,22 @@ require "inspec-bin/version"
 # inspec-core is a stripped-down version of the inspec gem with fewer exotic dependencies
 
 Gem::Specification.new do |spec|
-  spec.name          = "inspec-core-bin" # dallas multi-pass
+  spec.name          = "cinc-auditor-core-bin" # dallas multi-pass
   spec.version       = InspecBin::VERSION
-  spec.authors       = ["Chef InSpec Core Engineering "]
-  spec.email         = ["inspec@chef.io"]
+  spec.authors       = ["Chef InSpec Core Engineering", "Cinc Project"]
+  spec.email         = ["inspec@chef.io", "cinc-auditor@cinc.sh"]
   spec.summary       = "Infrastructure and compliance testing."
-  spec.description   = "InSpec executable for inspec-core gem. Use of this executable may require accepting a license agreement."
-  spec.homepage      = "https://github.com/inspec/inspec/tree/master/inspec-bin"
+  spec.description   = "Cinc Auditor (a community distribution of InSpec) executable for cinc-auditor-core gem."
+  spec.homepage      = "https://gitlab.com/cinc-project/auditor"
   spec.license       = "Apache-2.0"
 
   spec.require_paths = ["lib"]
   spec.required_ruby_version = ">= 2.4"
 
-  spec.add_dependency "inspec-core", "= #{InspecBin::VERSION}"
+  spec.add_dependency "cinc-auditor-core", "= #{InspecBin::VERSION}"
   spec.add_development_dependency "rake"
 
-  spec.files = %w{README.md LICENSE Gemfile} + ["inspec-core-bin.gemspec"] +
+  spec.files = %w{README.md LICENSE Gemfile} + ["cinc-auditor-core-bin.gemspec"] +
     Dir.glob("{lib,bin}/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
 
   spec.bindir = "bin"
-- 
2.11.0

